### ISSUE TITLE
[Bug] XRC custom property: uppercase in tag [sf#163]

### ISSUE THREAD

**Issue by DietmarSchwertberger** [DietmarSchwertberger/wxGlade#238]
**Reported by grumot on 2014-06-15 20:36 UTC**
Hello, 

It seems that with "custom widgets" and adding "extra properties for this widget", the XRC output changes the case of the first character of the property to an uppercase character. 

e.g.

~~~~~
wildcard          text (*.txt)|*.txt
~~~~~

is translated in the XRC file into:

~~~~~
<Wildcard>text (*.txt)|*.txt</Wildcard>
~~~~~

instead of:

~~~~~
<wildcard>text (*.txt)|*.txt</wildcard>
~~~~~

I am using this release (from debian):
0.6.8 on python 2.7.6 and wxPython 2.8.12.1

Thanks!
Phil

**Comment by DietmarSchwertberger** [comment]
**Updated by cgrohmann on 2014-06-23 20:08 UTC**
- **status**: open --> closed
- **assigned_to**: Carsten Grohmann

**Comment by DietmarSchwertberger** [comment]
**Commented by cgrohmann on 2014-06-23 20:08 UTC**
Hi Phil,

the bug is fixed in the revision 1412:17926443bae9 as well as the next official release.

Thank you for reporting this bug and for the detailed bug report incl. example.

Regards,
Carsten
